opts/envfile_test.go

```diff
@@ -29,7 +29,12 @@ func TestParseEnvFileGoodFile(t *testing.T) {
 
 _foobar=foobaz
 `
-
+	// Adding a newline + a line with pure whitespace.
+	// This is being done like this instead of the block above
+	// because it's common for editors to trim trailing whitespace
+	// from lines, which becomes annoying since that's the
+	// exact thing we need to test.
+	content += "\n    "
```

Comment:
Do we need a similar test, using a `tab` as whitespace?

Also, this looks to be adding *trailing* whitespace (the whitespace is appended to the content), not *leading* whitespace, is that correct?